packages/iov-ethereum/src/ethereumconnection.ts

```diff
@@ -740,14 +764,18 @@ export class EthereumConnection implements AtomicSwapConnection {
     return { ...transaction, fee: await this.getFeeQuote(transaction) };
   }
 
-  public async getSwaps(query: AtomicSwapQuery): Promise<ReadonlyArray<AtomicSwap>> {
+  public async getSwaps(
+    query: AtomicSwapQuery,
+    minHeight: number = 0,
+    maxHeight: number = Number.MAX_SAFE_INTEGER,
+  ): Promise<ReadonlyArray<AtomicSwap>> {
     if (isAtomicSwapIdQuery(query)) {
       const data = Uint8Array.from([...Abi.calculateMethodId("get(bytes32)"), ...query.swapid]);
 
       const params = [
         {
           to: constants.atomicSwapEtherContractAddress,
-          data: "0x" + Encoding.toHex(data),
+          data: `0x${Encoding.toHex(data)}`,
```

Comment:
We really should have a `toEthereumHex` helper. I saw and wrote this code too often now.

Reply:
I can add one, I'm also annoyed with it.